runtime/doc/vim_diff.txt

```diff
@@ -69,6 +69,8 @@ are always available and may be used simultaneously in separate plugins.  The
 
 See |nvim-intro| for a list of Nvim's largest new features.
 
+Enables |bracketed-paste-mode|.
+
```

Comment:
Maybe to contrast with vim:
 `Bracketed paste mode is built-in and enabled by default. |bracketed-paste-mode| is enabled.`